### PR TITLE
Run lint and format in CI

### PR DESCRIPTION
**Pull request by takker99**
close https://github.com/hayd/deno-udd/issues/41

### CODE REVIEW

File: main.ts [hunk 1]
```diff
@@ -48,11 +48,11 @@ Optional arguments:
 
 function version() {
   // FIXME this might be kinda a hacky way to do it...
-  let u = new DenoLand(import.meta.url);
+  const u = new DenoLand(import.meta.url);
   try {
     console.log(u.version());
-  } catch (err) {
-    console.log(undefined);
```
Comment (hayd) [inline main.ts:55]: hmmm. I wonder why I did this 😆

File: test.ts [hunk 2]
```diff
@@ -1,12 +1,6 @@
 import { udd, UddOptions, UddResult } from "./mod.ts";
 import { RegistryCtor } from "./registry.ts";
-import {
-  assert,
-  assertEquals,
-  assertThrowsAsync,
```
Comment (hayd) [inline test.ts:6]: a statement of intent 😆

**Comment by hayd** [comment]
Looks good, thanks again @takker99 !

File: test.ts (hunk 2 above)
Comment (takker99) [reply test.ts:6]: Sorry but I did nothing but follow `deno fmt`....
Why don't you add `// deno-fmt-ignore` above the line?

**Pull request merged**